app/controllers/apps_controller.rb

```diff
@@ -40,6 +38,13 @@ def destroy
   end
   private
 
+  def app_owner?
+    if !App.find(params[:id]).owners.include?(current_user)
```

Comment:
@jwpincus you can't extract the entire method itself, but we could extract the logic within the method. you'd need to write a method that takes in the id and user like so...

```
class App
  def self.check_owner?(id, user)
    find(id).owners.include?(user)
  end
end
```

then, in your method in your controller, you could have..

```
if !App.check_owner?(params[:id], current_user)
```